warm_transfer_flask/call.py

```diff
@@ -1,12 +1,12 @@
-from twilio.rest import TwilioRestClient
+from twilio.rest import Client
 from os import environ as ENV
 
 
 def call_agent(agent_id, callback_url):
     account_sid = ENV['TWILIO_ACCOUNT_SID']
     auth_token = ENV['TWILIO_AUTH_TOKEN']
     my_number = ENV['TWILIO_NUMBER']
-    client = TwilioRestClient(account_sid, auth_token)
+    client = Client(account_sid, auth_token)
     call = client.calls.create(to='client:' + agent_id, from_=my_number,
```

Comment:
@mosampaio [`to`](https://github.com/twilio/twilio-python/blob/380e6242bab91b7b9d155d28cd0b0d8233dd1bab/twilio/rest/api/v2010/account/message/__init__.py#L40) is a required parameter, using it as a named parameter is weird. You could extract the value to a variable naming it as what it is, like this:

```python
to = 'client: {}'.format(agent_id)
call = client.calls.create(to, from_=my_number, ...
```

In this way we are explicit about that `to` is a required parameter and the others are not.

What do you think?